### ISSUE TITLE
Encourage standardized budget format

### ISSUE THREAD

**Issue by Ainali** [datastory-org/the-version-controlled-political-party#1]
Instead of the csv with seemingly arbitrary columns, I think we could encourage some standardized format so that it becomes easier to compare different organizations to each other. Unfortunately, I don't know which ones are good, but perhaps @mattiasaxell who presented work on municipal budgets yesterday have ideas.

**Comment by lapidus** [comment]
Yes, great idea @Ainali. I didn't spend enough time yet on the individual CSV templates and standards. 
Happy for Pull Requests or links to existing standards.
